packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -309,6 +351,22 @@ export class AppStudioPluginImpl {
     return ok(await AppStudioClient.validateManifest(manifestString, appStudioToken!));
   }
 
+  public async migrateV1Project(ctx: PluginContext): Promise<any> {
+    let manifest: TeamsAppManifest | undefined;
+    const archiveAppPackageFolder = path.join(ctx.root, ArchiveFolderName, AppPackageFolderName);
+    const archiveManifestPath = path.join(archiveAppPackageFolder, V1_MANIFEST);
+    const newAppPackageFolder = path.join(ctx.root, AppPackageFolderName);
+    await fs.ensureDir(newAppPackageFolder);
+    if (await this.checkFileExist(archiveManifestPath)) {
+      await copyFiles(archiveAppPackageFolder, newAppPackageFolder, [V1_MANIFEST]);
```

Comment:
- Whose responsibility to copy V1 `appPackage` folder to `.archive`?
- There may be `appPackage.zip` and `README.md` under V1 `appPackage` folder, is it our expectation to also copy these two files?

Reply:
Good idea! We should exclude more files in `appPackage` folder. Maybe allowlist is a better way. I'm only worried about the changes of the run icon file name.


PS: There is another PR to archive the existing files in `core`. Here only contains the app studio part.

Reply:
Or we can exclude files according to the file extension

Reply:
I think we can simply exclude `appPackage.zip` and `README.md` as well since they are generated by V1. If user added some files in V1 `appPackage` folder, these files are out of our control, just let them here.